block_to_imp.c

```diff
@@ -177,7 +177,12 @@ static struct trampoline_set *alloc_trampolines(char *start, char *end)
 	}
 	metadata->buffers->headers[HEADERS_PER_PAGE-1].block = NULL;
 	mprotect(metadata->buffers->rx_buffer, PAGE_SIZE, PROT_READ | PROT_EXEC);
+#if defined(_WIN32) && (defined(__arm__) || defined(__aarch64__))
```

Comment:
Please can you implement the [`clear_cache`](https://github.com/gnustep/libobjc2/blob/226455bd10d8b4a0a090449b5b7ebf1e42daf0a5/block_to_imp.c#L25) macro (which is already target-specific) with the Windows version, rather than polluting generic code with target-specific ifdefs?